tests/test_streams.py

```diff
@@ -1,5 +1,5 @@
-from binance.streams import BinanceSocketManager
-from binance.client import AsyncClient
+from binance.ws.streams import BinanceSocketManager
```

Comment:
@pcriadoperez can't we expose this inside __init__ to avoid changing the import path?